### ISSUE TITLE
Can we unpin docutils?

### ISSUE THREAD

**Issue by nedbat** [executablebooks/sphinx-tabs#171]
### Describe the bug

I am trying to upgrade my development pins in coverage.py, and I get incompatible requirements:
```
Could not find a version that matches docutils<0.18,<0.20,<0.21,>=0.14,>=0.19,~=0.18.0 (from doc8==1.0.0->-r doc/requirements.in (line 10))
Tried: 0.3, 0.3.5, 0.3.7, 0.3.9, 0.4, 0.5, 0.6, 0.7, 0.8, 0.8.1, 0.9, 0.9.1, 0.10, 0.11, 0.12, 0.12, 0.13.1, 0.13.1, 0.14, 0.14, 0.15, 0.15, 0.15.1.post1, 0.15.2, 0.15.2, 0.16, 0.16, 0.17, 0.17, 0.17.1, 0.17.1, 0.18, 0.18, 0.18.1, 0.18.1, 0.19, 0.19
Skipped pre-versions: 0.14rc1, 0.14rc2, 0.16b0.dev0, 0.16b0.dev0, 0.16rc1, 0.16rc1, 0.17b1, 0.17b1, 0.17.1b1.dev0, 0.17.1b1.dev0, 0.18b1, 0.18b1, 0.18.1b0, 0.18.1b0, 0.19b1, 0.19b1
There are incompatible versions in the resolved dependencies:
  docutils<0.18 (from sphinx_rtd_theme==1.0.0->-r doc/requirements.in (line 14))
  docutils<0.21,>=0.19 (from doc8==1.0.0->-r doc/requirements.in (line 10))
  docutils~=0.18.0 (from sphinx-tabs==3.4.1->-r doc/requirements.in (line 15))
  docutils<0.20,>=0.14 (from sphinx==5.1.1->-r doc/requirements.in (line 12))
```
I know that docutils had some problems, but I'm wondering if sphinx-tabs still needs a strict requirement like `~0.18.0`.  I'm also asking other projects about loosening their requirements. I'm hoping we can all move to a more flexible set of requirements.

### Reproduce the bug

If you want to try the coverage.py upgrade, I can provide further instructions.

### List your environment

Mac OS, Python 3.7.13

**Comment by GlenNicholls** [comment]
Agreed, I ran into an issue with the following requirements:

```
Sphinx>=4.4.0 
sphinx-rtd-theme>=1.0.0
sphinx-tabs>=3.4.1
```

with this error:

```
The conflict is caused by:
    sphinx 4.4.0 depends on docutils<0.18 and >=0.14
    sphinx-rtd-theme 1.0.0 depends on docutils<0.18
    sphinx-tabs 3.4.1 depends on docutils~=0.18.0
```

**Comment by vcunat** [comment]
Some default to docutils 0.19 now, which also fails this condition though I haven't encountered an issue with the combination.

**Comment by JSS95** [comment]
Can be closed (#186)
